### PR TITLE
test(connect): Configure connect_test so that it uses its own ports

### PR DESCRIPTION
**Pull request by dustmop**

### CODE REVIEW

**Comment by dustmop** [comment]
I believe this will fix https://github.com/qri-io/qri/issues/1158, at least *some* of its failures. Should also finally fix https://github.com/qri-io/qri/issues/1159. May also fix https://github.com/qri-io/qri/issues/1047, though we haven't seen that failure in a while.

File: cmd/connect_test.go [hunk 1]
```diff
@@ -2,51 +2,42 @@ package cmd
...
+	defer run.Delete()
+
+	// Construct a mock registry to pass to the connect command
 	_, registryServer := regmock.NewMockServer()
 
-	path := filepath.Join(os.TempDir(), "qri_test_commands_connect")
-	t.Logf("temp path: %s", path)
+	// Configure ports such that other tests do not conflict with the connection ports
+	// TODO(dustmop): Add websocket.port to config, set that here
+	run.MustExec(t, "qri config set api.port 9871 rpc.port 9872")
```
Comment (b5) [inline cmd/connect_test.go]: I'd be awesome if we could set these to `0`, which would act as a sentinel for "choose any available port". If I'm not mistaken feeding the empty string to `net.Listen()` has this behaviour.

File: cmd/connect_test.go [hunk 2]
```diff
@@ -2,51 +2,42 @@ package cmd
...
+	run := NewTestRunner(t, "test_peer", "qri_test_connect")
+	defer run.Delete()
+
+	// Construct a mock registry to pass to the connect command
 	_, registryServer := regmock.NewMockServer()
 
-	path := filepath.Join(os.TempDir(), "qri_test_commands_connect")
-	t.Logf("temp path: %s", path)
+	// Configure ports such that other tests do not conflict with the connection ports
+	// TODO(dustmop): Add websocket.port to config, set that here
```
Comment (b5) [inline cmd/connect_test.go]: there's a partial-implementation of this in https://github.com/qri-io/qri/pull/1076, I'd love to pull off the `--no-p2p` part of that PR & revisit it.

Comment (dustmop) [reply cmd/connect_test.go]: I'll leave the TODO for now, but will definitely look into that to fix websockets soon.

**Pull request merged**